### PR TITLE
Postinstall script for npm 3 compat.

### PR DESCRIPTION
**Pull request by devel-pa**
This is the suggested way of symlinking the files.

Not tested.

### CODE REVIEW

**Comment by es128** [comment]
When you say "not tested", do you mean that you didn't try it locally, or that we still need to confirm it works on Windows, or what?

LGTM based on a cursory review, just need the corresponding change in the plugin to point to the new path in the `dist` directory.

File: postinstall.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ */
+
+var sysPath = require('path');
+var fs = require('fs');
+
+fs.mkdir('./dist');
+
+var runtime = ['', '.runtime'], amd = ['', '.amd'], min = ['', '.min'];
+
+for(var i = 0; i < runtime.length; i++) {
```
Comment (paulmillr) [inline postinstall.js]: can we use `forEach` here?

**Comment by devel-pa** [comment]
Hi guys, I'm on holiday and I'll answer slow. I will make the necessary changes and I'll try to test it locally ASAP.

**Comment by devel-pa** [comment]
Sorry for the double commit.
Tested under Windows and works.

**Comment by es128** [comment]
Looks great. Thanks @devel-pa

**Pull request merged**